MixedRemoteViewCompositor/Samples/LowLatencyMRC/UWP/Viewer/Viewer.csproj

```diff
@@ -95,10 +95,17 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="MixedRemoteViewCompositor, Version=255.255.255.255, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>$(SolutionDir)..\..\..\Build\$(Configuration)\Plugins\WSA\$(PlatformName)\MixedRemoteViewCompositor.winmd</HintPath>
+    <Reference Include="MixedRemoteViewCompositor">
+      <HintPath>..\..\..\..\Source\Win32\MixedRemoteViewCompositor.winmd</HintPath>

```

Comment:
modify the path to the repo's build folder "$(SolutionDir)..\..\..\Build\$(Configuration)\Plugins".